alica_engine/include/engine/PlanBase.h

```diff
@@ -58,7 +58,7 @@ class PlanBase
     const AlicaTime getLoopInterval() const;
     void setLoopInterval(AlicaTime loopInterval);
     void stop();
-    void start(const Plan* masterPlan, const IAlicaWorldModel* wm);
+    void start(const Plan* masterPlan, const Blackboard& globalBlackboard);
```

Comment:
Taking the global blackboard as a parameter here is strange. Isn't this just the `globalBlackboard` memeber variable?